### PR TITLE
COMP: Fix PythonQt wrapping of classes using nullptr

### PR DESCRIPTION
**Pull request by phcerdan**
This commit updates the python script ctkWrapPythonQt.py to check for nullptr.

Reference: https://github.com/Slicer/Slicer/pull/1102

### CODE REVIEW

**Comment by phcerdan** [comment]
Testing the wrapping locally along the nullptr changes

**Comment by jcfr** [comment]
Thanks. Let me know when your build complete.

**Comment by jcfr** [comment]
Suggested commit message:

```
COMP: Fix PythonQt wrapping of classes using nullptr

This commit updates the python script ctkWrapPythonQt.py to check for nullptr.

The script is used to parse C++ headers and extract list of derived Qt classes
for which a python module registering PythonQt decorator should be compiled.
```

**Comment by phcerdan** [comment]
The build works flawlessly!

**Comment by phcerdan** [comment]
Modify commit message, and add the same words as a docstring to the script.

File: CMake/ctkWrapPythonQt.py [hunk 1]
```diff
@@ -1,9 +1,15 @@
+""" ctkWrapPythonQt
+
+Script to parse C++ headers and extract list of derived Qt classes
```
Comment (jcfr) [inline CMake/ctkWrapPythonQt.py]: Thanks for adding the docstring :+1: 

It turns out I was incorrect in my initial suggestion. Here is an improve documentation:

```
CLI allowing to generate PythonQt decorator for constructor and destructor of classes.

The CLI expects one class per header file with a class name matching the filename. 

It generates the files `<namespace>_<target>.h` and `<namespace>_<target>_init.cpp`
where:
* `<namespace>` is a string like `org_commontk`
* `<target>` is a string like `CTKWidgets`
```

**Comment by phcerdan** [comment]
Added the more complete docstring to the script. Thanks @jcfr.

I think it is ready to be merged.

**Comment by jcfr** [comment]
Thanks. Closing. Integrated in https://github.com/commontk/CTK/commit/d11e259fd8a337ace09f5d99c0840d33b059e96d

**Pull request closed without merging**